homeassistant/components/deconz/binary_sensor.py

```diff
@@ -205,6 +227,8 @@ def async_add_sensor(_: EventType, sensor_id: str) -> None:
             ):
                 continue
 
+            async_update_unique_id(hass, sensor.unique_id, description)
```

Comment:
Wouldn't it make more sense to use the `homeassistant.helpers.entity_registry.async_migrate_entries()` helper?